src/sentry/web/urls.py

```diff
@@ -160,6 +161,8 @@ def init_all_applications():
         name='sentry-auth-link-identity'),
     url(r'^auth/login/$', AuthLoginView.as_view(),
         name='sentry-login'),
+    url(r'^auth/login/2fa/$', TwoFactorAuthView.as_view(),
```

Comment:
Fortunately, we don't currently have a user with the slug `2fa` or this would be problematic for them.